draft-irtf-cfrg-vdaf.md

```diff
@@ -3820,8 +3820,8 @@ All gadgets are listed in {{gadgets}}.
 
 | Parameter         | Value                                      |
 |:------------------|:-------------------------------------------|
-| `Valid`           | `Count(Field64)` (this section)            |
 | `Field`           | `Field64` ({{fields}})                     |
```

Comment:
nit: I think `field` should be lower case since it's lower case where we reference it
```suggestion
| `field`           | `Field64` ({{fields}})                     |
```